packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -104,8 +107,18 @@ function PostLink( {
 			<div className="edit-post-post-link__preview-link-container">
 				<ExternalLink
 					className="edit-post-post-link__link"
-					href={ postLink }
```

Comment:
`postlink` is coming from the `getCurrentPost` selector's `link` property, which in turn comes from the REST API ([see](https://developer.wordpress.org/rest-api/reference/posts/)). That field is [set](https://github.com/WordPress/wordpress-develop/blob/9d1a31cd0afed808e4fba9b296e1129e1c1b31ae/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1649) by the corresponding controller to use WordPress's [`get_permalink()`](https://developer.wordpress.org/reference/functions/get_permalink/) function.

It might be worth considering to modify _that_ function, in order to change the behavior for all cases where the permalink is used (including wp-admin Posts screen and Classic Editor), in order to get perfectly consistent behavior everywhere :thinking: